plugins/compilers/raspc-rasp/src/main/jflex/lexer.jflex

```diff
@@ -85,6 +85,8 @@ number = "-"?[0-9]+
 identifier = [a-zA-Z][a-zA-Z0-9]*
 label = {identifier}[":"]
 operator_constant = "="
+org=[\r|\n|\r\n]*"org"
+input=[\r|\n|\r\n]*"<input>"
```

Comment:
the same question... prefix `[\r|\n|\r\n]` is wrong.

Reply:
this was to enable empty lines before the directive, but probably should be handled in the grammar file